packages/sw-background-sync-queue/src/lib/background-sync-queue.js

```diff
@@ -76,13 +99,28 @@ class BackgroundSyncQueue {
 	 * @param {Object} input
 	 * @param {Request} input.request The request which is to be queued
 	 *
-	 * @returns {Promise} Promise which resolves when the request is pushed in
+	 * @return {Promise} Promise which resolves when the request is pushed in
 	 * the queue
 	 */
 	pushIntoQueue({request}) {
 		assert.isInstance({request}, Request);
 		return this._queue.push({request});
 	}
+
+	/**
+	 * This function is a call wrapper over `pushIntoQueue` used by higher
```

Comment:
Change 'This function is a call wrapper over' to 'Wraps the'. 

In reference materials, something are not complete sentences.